Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM quay.io/opendatahub/text-generation-inference:fast-283ec87
 
 USER root
 
+# Add grpc-ecosystem health probe
+# This is from https://github.ibm.com/ai-foundation/caikit-pt/blob/main/Dockerfile
+ARG GRPC_HEALTH_PROBE_VERSION=v0.4.15
```

Comment:
Can we switch it to latest, just in case this comes up with any security issue. The latest one seems to be `v0.4.19`